ros/src/pybullet_ros/plugins/control.py

```diff
@@ -14,13 +14,14 @@
 class pveControl:
     """helper class to receive position, velocity or effort (pve) control commands"""
 
-    def __init__(self, controller_type):
+    def __init__(self, controller_type, namespace):
         """constructor
         Assumes joint_name is unique, creates multiple subscribers to receive commands
         controller_type - position, velocity or effort
+        namespace - namespace of the robot
         """
         assert controller_type in ['position', 'velocity', 'effort']
-        rospy.Subscriber("/" + controller_type + '_controller/command',
+        rospy.Subscriber(namespace + controller_type + '_controller/command',
```

Comment:
same comment as in the JointStatesToFloat class

Reply:
Contrary to the JointStatesToFloat, I'm not doing `rospy.get_namespace()` but with `robot.get_namespace()` here and that should be okay because in the initialization of the robot, this happens:
```python
allowed = re.compile("[a-zA-Z0-9_]*$")
if not allowed.match(name):
    rospy.logerr(
        "[PyBulletRobot::init] Invalid name '{}' for a PyBulletRobot. Allowed characters are [a-zA-Z0-9_]. " +
        "Exiting now.".format(name))
    return
self._namespace = "/" + name + "/"
```